com.beeper.Beeper.yaml

```diff
@@ -0,0 +1,81 @@
+#This package is not affiliated to Beeper Team
+app-id: com.beeper.Beeper
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: startbeeper
+separate-locales: false
+finish-args:
+  # Version required to use the document-portal v4
+  - --require-version=1.7.0
+  # Required due to being a GUI application
+  - --socket=x11
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - --share=ipc
+  # Required for experimental wayland support
+  - --socket=wayland
+  # Required to provide Call functionality
+  - --socket=pulseaudio
+  - --device=all
+  # As a chat application, networking is required
+  - --share=network
+  # Required for appindicator in various desktop environments
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Required to allow screensaver/idle inhibition such as during video calls
+  - --talk-name=org.freedesktop.ScreenSaver
+  # Required for advanced input methods e.g. writing CJK languages
+  - --talk-name=org.freedesktop.portal.Fcitx
+  # Required to store access tokens (persistent logins)
+  - --filesystem=xdg-run/keyring
+  # Required for experimental wayland support
+  - --filesystem=xdg-run/pipewire-0
+  # For correct cursor scaling under Wayland
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.la
+  - /lib/*.a
+  - /share/pkgconfig
+  - /share/aclocal
+  - /share/gtk-doc
+  - /share/doc
+  - /share/info
+  - /share/man
+  - /man
+modules:
+  - name: beeper
+    buildsystem: simple
+    build-commands:
+      - chmod +x x64
+      - ./x64 --appimage-extract
+      - mv squashfs-root /app/bin/beeper
+      - install -D startbeeper -t /app/bin
+      - install -D com.beeper.Beeper.png /app/share/icons/hicolor/512x512/apps/com.beeper.Beeper.png
+      - install -D com.beeper.Beeper.desktop -t /app/share/applications/
+      - install -D $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.appdata.xml
+      - rm -rf x64
+    sources:
+      - type: script
+        dest-filename: startbeeper
+        commands:
+          - for i in {0..9}; do
+          - test -S $XDG_RUNTIME_DIR/discord-ipc-$i || ln -sf {app/com.discordapp.Discord,$XDG_RUNTIME_DIR}/discord-ipc-$i;
+          - done
+          - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+          - zypak-wrapper /app/bin/beeper/beeper "$@"
+
+      - type: file
+        only-arches: [x86_64]
+        url: https://download.beeper.com/linux/appImage/x64
```

Comment:
Unversioned urls :( Is a versioned one not available?

Reply:
Unfortunately no :(

Reply:
also is this redistributable?

Reply:
Yes it is, just confirmed with them

Reply:
> Unfortunately no :(

so the build will randomly break at the whim of the developers....


You should  setup flatpak-external-data-checker

https://github.com/flathub/flatpak-external-data-checker

Reply:
Great news! Apparently they use todesktop and people found a versioned link! (even if Beeper themselves told me they don't do it, so this should solve the issue)